boilerplate/fullstack/remix/app/routes/api.auth.$.tsx

```diff
@@ -0,0 +1,30 @@
+import { json } from "@remix-run/node";
+import { ActionFunctionArgs, LoaderFunctionArgs } from "react-router-dom";
+import handleAuthAPIRequest from "../lib/authAPIRequestHandler.js";
+import { createPreParsedRequest } from "../lib/superTokensHelpers.js";
+
+const handleCall = handleAuthAPIRequest(Response);
+
+// Action function for handling POST requests
+export async function action({ request }: ActionFunctionArgs) {
+    try {
+        const preParsedRequest = createPreParsedRequest(request);
+        const res = await handleCall(preParsedRequest);
```

Comment:
can the handleCall function itself not create the createPreParsedRequest in it? So here, and below, we would just do handleCall(request)

Reply:
✔️